scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_prep.sh
+#
+# This script processes the IMS input ascii/netcdf files onto the UFS model grid.
+# Inputs: IMS ascii/netcdf file, IMS index file (is model resolution specific) 
+#         and model restart files.
+# Outputs: file with i) IMS-derived snow cover fraction over land on UFS model 
+#         grid, and ii) snow depth derived form the IMS snow cover fraction, 
+#         using an inversion of the noah model snow depletion curve.
+#
+################################################################################
+#  Utilities
+RES=${CASE:1}
+
+#  Directories.
+export gdasapp_dir=${gdasapp_dir:-"${HOMEgfs}/sorc/gdas.cd/"}
+BKGDIR=${DATA}/bkg
+
+#  Filenames
+export IMS_IODA=${IMS_IODA:-"${HOMEgfs}/sorc/gdas.cd/build/bin/imsfv3_scf2ioda.py"}
+export CALCFIMS=${CALCFIMS:-"${HOMEgfs}/sorc/gdas.cd/build/bin/calcfIMS.exe"}
+
+export OROGTYPE="C${RES}_oro_data"
+FILEDATE=${PDY}.${cyc}0000
+DOY=$(date +%j -d "${PDY} + 1 day")
+YYYY=${PDY:0:4}
+ntiles=6
+################################################################################
+# IMS proc
+if [[ ${cyc} == "18" ]]; then
+  WORKDIR=${DATA}/obs
+  cd "${WORKDIR}" || exit 99
+  if [[ -e fims.nml ]]; then
+    rm fims.nml
+  fi
+
+cat >> fims.nml << EOF
+ &fIMS_nml
+  idim=${RES}, jdim=${RES},
+  otype=${OROGTYPE},
+  jdate=${YYYY}${DOY},
+  yyyymmddhh=${PDY}.${cyc},
+  imsformat=2,
+  imsversion=1.3,
+  IMS_OBS_PATH="${WORKDIR}/",
+  IMS_IND_PATH="${WORKDIR}/"
+  /
+EOF
+
+  # stage restarts
+  for tile in $(seq 1 "${ntiles}"); do
+    if [[ ! -e ${FILEDATE}.sfc_data.tile${tile}.nc ]]; then
+      ${NCP} "${BKGDIR}/${FILEDATE}.sfc_data.tile${tile}.nc" "${WORKDIR}"
+    fi
+  done
+
+  # stage observations
+  ${NCP} "${COM_OBS}/gdas.t${cyc}z.ims${YYYY}${DOY}_4km_v1.3.nc" "${WORKDIR}/ims${YYYY}${DOY}_4km_v1.3.nc"
```

Comment:
Please do not hard-wire `RUN` variable.
```suggestion
  ${NCP} "${COM_OBS}/${RUN}.t${cyc}z.ims${YYYY}${DOY}_4km_v1.3.nc" "${WORKDIR}/ims${YYYY}${DOY}_4km_v1.3.nc"
```

Reply:
Done the changes.